brownie/README.md

```diff
@@ -0,0 +1,12 @@
+# Brownie
+
+Brownie is a housekeeping daemon wose repsonsibility is to find and remove stale resources which were:
```

Comment:
Minor typo. Change `wose` to `whose`

Reply:
And 'repsonsibity' to 'responsibility'.

Reply:
The readme was updated, please have a quick read again.